### DISCUSSION TITLE
坚果云同步，轮询服务器间隔设置为1到10，总是正常运行一会儿了，提示失败

### DISCUSSION THREAD

**Discussion by IZJ777** [Jeric-X/SyncClipboard#106] [Q&A]
老师们，我用的坚果云同步，然后轮询服务器间隔我设置的5，开始windows端和ios端都一切正常，然后过一会儿。ios就一直提示Failed to connect to server
![Clip_20240717_005820](https://github.com/user-attachments/assets/ce9f8ff0-b376-42b0-92bc-ff30414d5306)

电脑一直提示剪切板下载失败Response status code does not indicatesuccess:503(Service Unavailable)。
![Clip_20240717_005833 (1)](https://github.com/user-attachments/assets/5c8e0c51-c940-4d5a-a750-c2d232999b8c)

搞了大半天了 轮询服务器间隔和最大上传文件大小 值我也改了。一直这样子不知道怎么办。想请教各位大佬正确的操作方式，谢谢

**Comment by Jeric-X** [comment]
坚果云是这样的，服务器对访问次数限制的很严格，已经不适合作为自动同步的服务器了，非要使用坚果云时最好不要在任何地方开启自动同步，永远使用手动同步。
非要自动同步的话，当前iOS端内置的轮询时间是1秒，也是可以手动改的，桌面端和iOS都改成10秒，应该能多坚持一会

**Comment by IZJ777** [comment]
那有没有推荐的自动同步服务器哇，我看你gt里边nextcloud，alist这些都需要魔法才能快速访问，所以作为同步服务器延迟比较高。
发自iZJ

**Reply by long45343** [reply]
> 那有没有推荐的自动同步服务器哇，我看你gt里边nextcloud，alist这些都需要魔法才能快速访问，所以作为同步服务器延迟比较高。 发自iZJ
> […](#)
> 在 2024年7月17日，08:32，JericX ***@***.***> 写道： ﻿ 坚果云是这样的，服务器对访问次数限制的很严格，已经不适合作为自动同步的服务器了，非要使用坚果云时最好不要在任何地方开启自动同步，永远使用手动同步。 非要自动同步的话，当前iOS端内置的轮询时间是1秒，也是可以手动改的，桌面端和iOS都改成10秒，应该能多坚持一会 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.

你可以自己找一下那种几块钱的国内小云服务器，NAT机器都可以（这里就不做推荐了），然后自己搭建一个，这样效果还可以

**Comment by long45343** [comment]
坚果云不适合自动同步这玩意，次数太高了
